arm/proofs/decode.ml

```diff
@@ -296,11 +296,11 @@ let decode = new_definition `!w:int32. decode w =
     else if q then
       let pos = (val imm4) * 8 in
       // datasize is fixed to 128.
-      SOME (arm_EXT (QREG' Rd) (QREG' Rm) (QREG' Rn) pos)
+      SOME (arm_EXT (QREG' Rd) (QREG' Rn) (QREG' Rm) pos)
     else NONE
 
   | [0:1; q; 1:1; 0b011110:6; immh:4; abc:3; cmode:4; 0b01:2; defgh:5; Rd:5] ->
-    // MOVI, USRA (Vector)
+    // MOVI, USRA (Vector), SLI (Vector)
```

Comment:
This SLI (Vector) instruction's bitpattern again overlapped with MOVI (and USRA (Vector)). Luckily our new `bm_check_disjointness` in HOL Light's bitmatch.ml successfully catched this and pointed out where to fix! :)